src/main/java/org/rundeck/plugins/S3LogFileStoragePlugin.java

```diff
@@ -390,8 +383,7 @@ public void storeMultiple(final MultiFileStorageRequest files) throws IOExceptio
                     MultiFileStorageRequestErrors errors = (MultiFileStorageRequestErrors) files;
                     errors.storageFailureForFiletype(filetype, e.getMessage());
                 } else {
-                    logger.log(Level.SEVERE, e.getMessage());
-                    logger.log(Level.FINE, e.getMessage(), e);
+                    logger.error(e.getMessage());
```

Comment:
I wouldn't remove the FINE log entry. If its already there then its probably helpful for debugging problems.